src/containerapp/azext_containerapp/custom.py

```diff
@@ -1305,6 +1323,12 @@ def containerapp_up(cmd,
         if app.get()["properties"]["provisioningState"] == "InProgress":
             raise ValidationError("Containerapp has an existing provisioning in progress. Please wait until provisioning has completed and rerun the command.")
 
+    if is_azureml_app:
+        env, workload_profile_name, app_cpu_limit, app_memory_limit = _validate_azml_env(cmd, app, env, environment, resource_group, resource_group_name, workload_profile_name)
```

Comment:
You might create env in `_validate_azml_env`, right?
So this function is not only for validation. It might create some resources.
Recommend to show this information in the function name.